DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -16,7 +16,10 @@
   <class name="pat::Lepton<reco::BaseTau>" />
 
   <!-- PAT Objects, and embedded data  -->
-  <class name="pat::Electron"  ClassVersion="33">
+  <class name="pat::Electron"  ClassVersion="36">
+   <version ClassVersion="36" checksum="3720919820"/>
+   <version ClassVersion="35" checksum="905410242"/>
+   <version ClassVersion="34" checksum="1857282274"/>
```

Comment:
@slava77  
actually in the newly created area I couldn't compile by just incrementing class version by +1.
it gets compiled with version 35 for pat::Electron and version 20 for pat::Photon
while in the release we have version 33 for pat::Electron and 18 for pat::Photon.
Should I do a commit back to these versions?